src/features/events_list/atoms/eventListResource.ts

```diff
@@ -20,8 +20,8 @@ export const eventListResourceAtom = createResourceAtom(
       bbox: deps?.filters.bbox?.join(','),
     };
 
-    const responseData = await apiClient.get<Event[]>('/events/', params, true);
-    if (responseData === undefined) throw new Error('No data received');
+    const responseData =
+      (await apiClient.get<Event[]>('/events/', params, true)) ?? [];
```

Comment:
maybe it's a good idea to process undefined also by `|| [];` ?

Reply:
It depends on which way you look at it. In terms of defensive programming it is better to check for all invalid values including type (for example number 1 will pass the || check), and this how we usually do in pure js.
in terms of performance and predictable behavior it is better to keep types correct and be specific about what the code does, it's more typescript way